firstHomework/primeNumbers/main.c

```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
+#include <math.h>
+#include <stdbool.h>
+
+bool isPrimeNumber(const int number)
+{
+    for (size_t i = 2; i <= pow(number, 0.5); ++i)
```

Comment:
```suggestion
    for (size_t i = 2; i * i <= number; ++i)
```
Это то же самое, но без вызова довольно трудоёмкой функции pow